api/config/config_test.json

```diff
@@ -140,6 +140,6 @@
     "clientId": "e5c5d714-d597-48c8-b564-a249d729d0c9"
   },
   "accessRenewal": {
-    "expiryDays": 365
+    "expiryDays": 3650
```

Comment:
Is this too far out to be reasonable? What about 1080 (three years)? Nevermind, it probably doesn't matter.

Reply:
My concern is:
We have deps that haven't been touched in years. One dat three years from now the test env gets obliterated because we never addressed the expiry issue, and all the tickets got lost in the void.